docs/en/Community-Articles/2024-05-07-using-fluent-validation-with-abp/post.md

```diff
@@ -0,0 +1,95 @@
+# Using FluentValidation with ABP
+
+## What is Validation? Why Do We Need to Validate?
+Validation is checking whether a data is valid or not. We can liken validations to the cell membrane of our application. Just as the cell does not want to let in harmful substances, we do not want to add erroneous data to the database, which is critical to our application. Validations allow data that follows the rules to reach the database. Data that does not comply with the rules will not access the database at all, and the operation will fail.
+
+![validation](./images/validation.png)
+
+Validations make your application run more efficiently because they are not connected to the database.  Validations should be applied in different layers of applications. Validations are applied on the UI side. Validations should also be applied on the backend side and database to prevent malicious users from passing these validations. Fluent validation is often used to create validations on the backend side.
+
+## What is Fluent Validation?
+Fluent validation is a library for checking whether data is valid or not. Fluent validation can be applied to your code in a fluent and understandable way.
+
+
+## Why We Should Use Fluent Validation?
+Fluent Validation allows you to define your validation rules in a clear and flexible way. This means you can comfortably handle complex validation scenarios in your code. This makes your development process much more manageable. The readability that Fluent Validation offers really makes things easier. Having a clear understanding of what your validation rules do is a huge advantage when working on your code. In short, your code is cleaner and clearer. Fluent Validation is also very functional in terms of testability. By defining your validation rules in separate classes, you can easily test and maintain these rules. Fluent Validation is a widely used validation library on the .NET platform. As such, it has become a common standard among .NET developers. This provides advantages in terms of community support and compatibility. So, using Fluent Validation simplifies your development process by making your code more understandable, manageable and testable.
+
+## How to Use Fluent Validation in ABP?
+![fluent validation](./images/abp-fluentvalidation.png)
+
+Using Fluent validation with Abp is quite simple. Open a command line window in the folder of the project (.csproj file) and type the following command: 
+abp add-package Volo.Abp.FluentValidation
```

Comment:
````suggestion
Using Fluent validation with ABP is quite simple. Open a command line window in the directory of your project (.csproj file) and type the following command: 

```bash
abp add-package Volo.Abp.FluentValidation
```

````

Also, here, specify in which project's directory this command needs to be executed because, in this article, you are using a layered application.